doc/source/tutorial/stats/continuous_nakagami.rst

```diff
@@ -42,10 +42,10 @@ The probability density function of the :code:`nakagami` distribution in SciPy i
     f(x; \nu, \mu, \sigma) = 2 \frac{\nu^\nu}{ \sigma \Gamma(\nu)}\left(\frac{x-\mu}{\sigma}\right)^{2\nu - 1} \exp\left(-\nu \left(\frac{x-\mu}{\sigma}\right)^2 \right),\tag{1}
     \end{equation}
 
-for :math:`x` such that :math:`\frac{x-\mu}{\sigma} >= 0`, where :math:`\nu \geq \frac{1}{2}` is the shape parameter,
+for :math:`x` such that :math:`\frac{x-\mu}{\sigma} \geq 0`, where :math:`\nu \geq \frac{1}{2}` is the shape parameter,
 :math:`\mu` is the location, and :math:`\sigma` is the scale.
 
-The log-likelihood function is therefore
+The log-likelihood function is therefore:
```

Comment:
```suggestion
The log-likelihood function is therefore
```
Not a big deal, but the original doesn't have colons because the equation is considered part of the sentence, and no punctuation would be appropriate if the equation were expressed as words.
Thanks for fixing the `\geq`s, though! `>=` is what I would type into code or MS Word Equation editor; sometimes I forget that I'm using LaTeX!

Reply:
> Not a big deal, but the original doesn't have colons because the equation is considered part of the sentence, and no punctuation would be appropriate if the equation were expressed as words.

I see: I'll remove the extra-punctuation than I've added.

>  Thanks for fixing the \geqs, though! >= is what I would type into code or MS Word Equation editor; sometimes I forget that I'm using LaTeX!

No problems. 😉

Reply:
Great! We'll give others a few days to review. I'll merge on Monday or so unless there are new things that come up.